src/fsharp/FSharp.Compiler.Service.ProjectCracker/ProjectCracker.fs

```diff
@@ -33,10 +31,10 @@ type ProjectCracker =
         arguments.Append(' ').Append(enableLogging.ToString()) |> ignore
         for k, v in properties do
             arguments.Append(' ').Append(k).Append(' ').Append(v) |> ignore
-
+        let codebase = Path.GetDirectoryName(Uri(typeof<ProjectCracker>.Assembly.CodeBase).LocalPath)
```

Comment:
I'm a little unclear on the difference here. Does one or the other work even in the presence of shadow-copying?